Noisily sorted.txt

```diff
@@ -0,0 +1,5 @@
+1
+N true RANDOM true 56 71 0 1
+N false SORTED 363 125 1 -1
```

Comment:
why do you have to specify coordinates?

Reply:
Because that's how it knows where to display them when you open the editor

Reply:
guess this also depends on the discussion below; it would be nice to have a "default organization" if you just type it up

Reply:
That's an interesting idea